packages/metal-quartz-link/src/MetalQuartzLink.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import Component from 'metal-component';
+import MetalQuartzIcon from 'metal-quartz-icon';
+import Soy from 'metal-soy';
+import { core, object } from 'metal';
+import { validators } from 'metal-state';
+
+import templates from './MetalQuartzLink.soy.js';
+
+/**
+ * Implementation of the Metal Quartz Link.
+ */
+class MetalQuartzLink extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	created() {
+		MetalQuartzLink.instances.push(this);
```

Comment:
Hey Guys, What I want to accomplish here is to expose the component to the outside world so that, lets say, someone using jQuery has access to the state without having to search through the window object. It would be nice to have some kind of method that returns a metal component on the page based on a selector, dom element, or some kind of attribute.